src/internal/components/dropdown/dropdown-fit-handler.ts

```diff
@@ -141,13 +158,13 @@ export const getWidths = ({
   stretchBeyondTriggerWidth?: boolean;
 }) => {
   // Determine the width of the trigger
-  const triggerWidth = triggerElement.getBoundingClientRect().width;
+  const { inlineSize: triggerWidth } = getLogicalBoundingClientRect(triggerElement);
```

Comment:
Same here: can we make the variable names also logical so they stay up to date with their actual meaning?

Reply:
Updated!